src/crypto-api/CryptoEvent.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2024 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum CryptoEvent {
+    /**
+     * Fires when the trust status of a user changes.
+     * @param {string} userId - the user whose trust status changed
+     * @param {UserVerificationStatus} userTrustLevel - the new trust level
+     */
+    UserTrustStatusChanged = "userTrustStatusChanged",
+    /**
```

Comment:
would be nice to add some blank lines between the entries to help with readability